logicrunner/artifacts/instrument.go

```diff
@@ -18,9 +18,10 @@ package artifacts
 
 import (
 	"context"
-	"github.com/insolar/insolar/insolar/flow"
 	"time"
 
+	"github.com/insolar/insolar/insolar/flow"
+
```

Comment:
strange changes, if you want to do it properly - first block - inner packages, second block - outer packages, third . block - insolar packages